apps/admin-gui/src/app/facilities/pages/facility-detail-page/facility-service-status/facility-task-results/facility-task-results.component.ts

```diff
@@ -36,23 +37,17 @@ export class FacilityTaskResultsComponent implements OnInit {
 
   ngOnInit(): void {
     this.loading = true;
-    this.route.parent.params.subscribe(parentParams => {
-      this.facility = {
-       id: parseInt(parentParams['facilityId'], 10),
-       beanName: "Facility"
-      };
+    this.facility = this.entityStorageService.getEntity();
```

Comment:
remove from the *ngIf the "loading" condition from the buttons and the title of the page